config/candidate-api.yml

```diff
@@ -155,6 +155,31 @@ components:
          format: date-time
          description: Time of last change
          example: 2021-05-20T12:34:00Z
+        application_status:
```

Comment:
All good on swagger 👌

Reply:
TIL YAML doesn't mind how many spaces you use to indent, as long as you use at least one and you're consistent 🙃

https://docs.saltproject.io/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html#indentation

Reply:
That said, I think it would be neater to standardise on 2 throughout this file?

Reply:
Yea have sorted 👍